test/Interop/Cxx/templates/function-template-typechecker-errors.swift

```diff
@@ -25,12 +25,12 @@ public protocol A { }
 public protocol B { }
 public protocol C { }
 
-// CHECK: error: unexpected error produced: could not generate C++ types from the generic Swift types provided; the following Swift type(s) provided to 'passThrough' could not be converted: any A & B
+// CHECK: function-templates.h{{.*}}: error: diagnostic produced elsewhere: could not generate C++ types from the generic Swift types provided; the following Swift type(s) provided to 'passThrough' could not be converted: any A & B
```

Comment:
You should be able to match the line number with `[[@LINE+1]]`

Reply:
This location is in a header that we import, so the line reported here has no relationship to the lines in this file. Or is there a way to make that work regardless?

Reply:
Oh year, right! It could be done by putting the CHECK line in the header, and passing the header to FileCheck instead. But that may or may not be suitable here.